app/javascript/app/components/info-icon/info-icon-component.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import infoIcon from 'assets/icons/info.svg';
+import styles from './info-icon-styles.scss';
+
+class InfoIcon extends PureComponent {
+  // eslint-disable-line react/prefer-stateless-function
+  render() {
+    const { toggleInfo, infoOpen, right, noPadding } = this.props;
+    return (
+      <Button
+        className={cx(styles.btnInfo, {
+          [styles.btnInfoActive]: infoOpen,
+          [styles.right]: right,
+          [styles.noPadding]: noPadding
+        })}
+        onClick={toggleInfo}
+      >
+        <Icon icon={infoIcon} />
+      </Button>
+    );
+  }
+}
+
+InfoIcon.propTypes = {
```

Comment:
We don't need all of this props as we could pass our custom className to customize the styles.